presentation/src/main/java/me/androidbox/presentation/BusbyTaskyApplication.kt

```diff
@@ -1,7 +1,45 @@
 package me.androidbox.presentation
 
 import android.app.Application
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Build
 import dagger.hilt.android.HiltAndroidApp
 
 @HiltAndroidApp
-class BusbyTaskyApplication : Application()
\ No newline at end of file
+class BusbyTaskyApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        createNotificationChannel(generateChannels())
+    }
+
+    private fun createNotificationChannel(listOfChannel: Map<String, String>) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            listOfChannel.map { mapOfChannel ->
+                val notificationManager = getNotificationManager()
+                val notificationChannel = NotificationChannel(
+                    mapOfChannel.key,
+                    mapOfChannel.value,
```

Comment:
This is correct

But
When you check the key and values on the map, they are the other way wrong

Key should be "event", "reminder", "task", and value should have the description
But on the map creation, you are doing it the opposite way

Reply:
@MKiperszmid Thanks for catching that. I have updated